app/views/arts/show.html.erb

```diff
@@ -0,0 +1,10 @@
+<h1><%= @art.name %></h1>
+
+<ul>
+  <li><%= @art.location %></li>
+  <li><%= @art.description %></li>
+  <% unless @art.artist.empty? %>
```

Comment:
I might use an if statement a little easier to read... `if @art.artist.present?`